### DISCUSSION TITLE
Welcome to sanity-plugin-taxonomy-manager Discussions!

### DISCUSSION THREAD

**Discussion by andybywire** [andybywire/sanity-plugin-taxonomy-manager#31] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kjkaber-EIS** [comment]
Hello!
I've been playing around with the taxonomy manager plugin and see all sorts of applications for it. It's great!

I do have one question - is it possible to add fields or manipulate fields for the Concept documents? For example, adding an image, approval flag, and/or other document lookups. I can create these fields in a new document/schema, but unsure where in the plugin files I'd make adjustments (or if I can without breaking anything).

Thanks again!
k

**Reply by andybywire** [reply]
Hi @kjkaber-EIS,
Thanks for the note—I’m glad to hear you’re finding the plugin useful!
Regarding your question: there is not currently a way to add custom fields the the `skosConcept` or `skosConceptScheme` document types from the plugin itself. If you want to add custom fields to these, I suspect the way to do that now would be to fork your own version, integrate it with your studio code, and add fields to the documents in your forked version. I haven’t tried this personally, but this should allow you to continue to apply updates from this repo as well. 
That said, a “documents tagged with this concept” view is one I’d love to get in here sooner rather than later. If that’s something you’re interested in taking on, please do feel free to submit a PR. Revision/approval workflows may also be something that I eventually integrate, but that’s realistically a ways off. 
Either way, thanks for trying the plugin out—and please do let me know how it goes as you continue to work with it. 
Best,
Andy
